content/200-concepts/100-components/02-prisma-client/030-crud.mdx

```diff
@@ -450,7 +450,7 @@ The following [`findFirst`](/reference/api-reference/prisma-client-reference#fin
       id: "desc"
     }
   })
-}
+})
```

Comment:
Isn't it more the opposite, that the `}` is "too much" and should just be removed?
```suggestion
```

(The whole code example is indented by 2 spaces, which could of course also be fixed)

Reply:
agree!